docs/source/roadmap.rst

```diff
@@ -60,9 +66,19 @@ Submit a PR!
 Release History
 ---------------
 
+.. rubric:: Done in v2.9
+
+* **Orquesta Second Beta:** Second beta of new "Orquesta" workflow engine.
+* **WebUI:** Real-time streaming output, and Inquiries support
+* **Action Output Structure Definition:** Enable optional definition of action payload, so that it
+  can be inspected and used when passing data between actions in workflows.
+* **ChatOps:** Support for Microsoft Teams.
+* **k8s:** Reference configuration for running |st2| Enterprise on k8s.
```

Comment:
Need to include that K8s Helm Chart is focused on specific environment to run StackStorm  cluster in a `HA` mode.
Also please note that we plan to ship very limited _BETA_ version with `2.9` and stable with `3.0`.